### ISSUE TITLE
Official terraform provider support

### ISSUE THREAD

**Issue by mooreds** [FusionAuth/fusionauth-issues#1089]
### Description

We have community support for a terraform provider: https://github.com/FusionAuth/fusionauth-issues/issues/576

But I'd like to gauge community interest for an official supported provider. 

Please upvote if this is important to you.

**Comment by davelesser** [comment]
As the creator of a community-supported open source Terraform provider for FusionAuth[0][1], GPS Insight would be happy transition ours to you, if that's helpful. :grinning: 

[0] https://github.com/gpsinsight/terraform-provider-fusionauth
[1] https://registry.terraform.io/providers/gpsinsight/fusionauth/latest

**Comment by robotdan** [comment]
Yes, I think that is what we had in mind...  is to take your excellent work and just provide some official support for it and ease your burden. 

We really appreciate your contribution to the community!!

**Comment by tmatilai** [comment]
Are there any progress on this?

Unfortunately the community provider seems abandoned and hasn't had any updates for a couple of months, and [is even broken](https://github.com/gpsinsight/terraform-provider-fusionauth/issues/144) with the latest FA versions. 🙁

**Comment by robotdan** [comment]
To provide some background - @davelesser and his team have graciously been building and supporting the Terraform provider for the past few years, and we are very grateful for their contribution. 

We are in the process of transitioning the project from the current maintainer. In the mean time we can help with PRs if there is anything that needs to be moved along. 

There is an open PR for the issue you mentioned, and I think we are getting ready to merge that one, I just had a few concerns with breakage and @MCBrandenburg was nice enough to lend a hand. 
- https://github.com/gpsinsight/terraform-provider-fusionauth/pull/145

**Comment by VeeeneX** [comment]
The PR was merged and as @tmatilai pointed out, https://github.com/gpsinsight/terraform-provider-fusionauth seems to be abadonned? currently not working with latest 1.40.2 for general configurations, so probably now is the right time to take the ownership and publish the official terraform provider for FA. 

eg:
```
╷
│ Error: json: Unmarshal(nil)
│ 
│   with fusionauth_email.forgot_password,
│   on tenant.tf line 23, in resource "fusionauth_email" "forgot_password":
│   23: resource "fusionauth_email" "forgot_password" {
│ 
╵
╷
│ Error: json: Unmarshal(nil)
│ 
│   with fusionauth_user_action.lock_user,
│   on tenant.tf line 263, in resource "fusionauth_user_action" "lock_user":
│  263: resource "fusionauth_user_action" "lock_user" {
│ 
╵
```

It would be awesome to have a official provider! :relaxed:

**Comment by mooreds** [comment]
I am going to close this out, since we have official terraform support and we release it regularly.

Here's the provider: https://github.com/fusionauth/terraform-provider-fusionauth

Here's our docs: https://fusionauth.io/docs/operate/deploy/terraform

If you see gaps in this provider, please file them in that repo.
